### ISSUE TITLE
Error installing because of jags

### ISSUE THREAD

**Issue by mikeyEcology** [BrandonEdwards/bbsBayes#198]
Hi, Thank you for creating this package. I'm trying to install from CRAN and from github in R 4.2.0 on a Ubuntu machine (using):
```
install.packages("bbsBayes")
# when that didn't work I tried
devtools::install_github("BrandonEdwards/bbsBayes")
```
I get an error about jags installation. I don't use jags, I use Stan, so I'm not sure if there is something special I need to do. I am really only trying to use the `fetch_bbs_data` function, so I could just remake that function if installation is not possible, but I'm hoping to avoid that. 

Error message:
>Error: .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/usr/local/lib/R/site-library/rjags/libs/rjags.so':
  libjags.so.4: cannot open shared object file: No such file or directory
Execution halted
ERROR: lazy loading failed for package ‘bbsBayes’
removing ‘/usr/local/lib/R/site-library/bbsBayes’
The downloaded source packages are in
        ‘/tmp/Rtmp9j15kX/downloaded_packages’
Warning message:
In install.packages("bbsBayes") :
  installation of package ‘bbsBayes’ had non-zero exit status

**Comment by BrandonEdwards** [comment]
Hi there! Was actually just trying to figure this issue out with someone else the other day. I have to look into it a bit more, but something changed in R 4.2 and above that messed with a bunch of R packages, including rjags. The developers of JAGS are aware of this and have released JAGS 4.3.1. See this blog post for more info: https://martynplummer.wordpress.com/2022/04/12/windows-update-jags-4-3-1-is-released/

In any case, would recommend installing JAGS 4.3.1, then try installing bbsBayes, and see if that works?

B

**Comment by AdamCSmithCWS** [comment]
Yes, the Stan version of bbsBayes is well on its way. Should be available early in the new year. 

Also, if you're just looking for the bbs data download functions, you could consider the [bbsAssistant ](https://github.com/trashbirdecology/bbsAssistant) package as well

**Comment by mikeyEcology** [comment]
Thanks so much for your swift responses @BrandonEdwards and @AdamCSmithCWS! I did install jags first but still got that error. 
Since I'm not planning to use jags, I created a [fork](https://github.com/mikeyEcology/bbsBayes) of this repo that just removes the requirements for jags from the namespace and description (obviously this wouldn't work for any of the Bayes modeling). I might checkout bbsAssistant, but so far I'm really appreciating your functions for downloading and working with the data.
